Controllers/ExampleController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Bolt\Extension\YourName\ExtensionName\Controllers;
+
+use Bolt\Application;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+class ExampleController
+{
+    /**
+     * Bolt Application instance
+     *
+     * @var Application
+     */
+    private $app;
+
+    /**
+     * Extension configuration
+     *
+     * @var array
+     */
+    private $config;
+
+    function __construct(Application $app, $config)
+    {
+        $this->app = $app;
+        $this->config = $config;
+    }
+
+    /**
+     * Handles GET requests on /example/url/in/controller
+     */
+    public function exampleUrl()
+    {
+        // do something
+    }
+
```

Comment:
Controller routes are passed the request obejct, and also must return something, a better example of both might be:
```php
    /**
     * Handles GET requests on /example/url/in/controller
     *
     * @param Request $request
     * 
     * @return Response
     */
    public function exampleUrl(Request $request)
    {
        $response = new Response('Hello, World!', Response::HTTP_OK);
        
        return $response;
    }
```